_config.yml

```diff
@@ -35,9 +35,9 @@ tierthreeplan: Tiny Cloud Professional Plan
 enterpriseplan: Tiny Custom Plans
 ## common tiny.cloud URLs
 supporturl: https://support.tiny.cloud
-plugindirectory: https://www.tiny.cloud/apps/
+plugindirectory: https://www.tiny.cloud/tinymce/features/
 gettiny: https://www.tiny.cloud/get-tiny
-accountpageurl: https://apps.tiny.cloud/my-account
+accountpageurl: https://www.tiny.cloud/my-account
```

Comment:
This doesn't look correct, as it's immediately redirected to `https://www.tiny.cloud/my-account/` so shouldn't it be that instead?

Reply:
I'll check where we use this attribute to make sure it wont break anything. typically we use these URLs as `{{site.attribute}}/page/` or `{{site.attribute}}` and let it redirect to the slash version

Reply:
ahh, right I forgot about that sorry :cry:

Reply:
no worries. I'll leave as is. (it's easier to read in text as-is)